noxfile.py

```diff
@@ -31,3 +32,25 @@ def default(session):
 def unit(session):
     """Run the unit test suite."""
     default(session)
+
+
+@nox.session(python="3.7")
```

Comment:
Not 3.8?

Reply:
Since 3.8 works just as good, sure, why not. Done.